src/ngraph/runtime/backend_manager.cpp

```diff
@@ -83,7 +83,7 @@ shared_ptr<runtime::Backend> runtime::BackendManager::create_backend(const std::
     if (it != registry.end())
     {
         new_backend_t new_backend = it->second;
-        rc = shared_ptr<runtime::Backend>(new_backend(config.c_str()));
+        backend = new_backend(config.c_str());
```

Comment:
Is there any reason not to return a `std::unique_ptr<runtime::Backend>` directly from the `new_backend()` call?  (We could pass in `config` as a string, too.)

Reply:
Since new_backend is a C function it shouldn't return c++ objects. Pointers are ok.